tests/gdk/common/test_utils.py

```diff
@@ -181,3 +181,15 @@ def test_cli_version_check_latest_available(mocker):
 )
 def test_get_next_patch_version(version):
     assert utils.get_next_patch_version(version) == "1.0.1"
+
+
+def test_valid_recipe_size(mocker):
+    mocker.patch("os.path.getsize", return_value=1000)
+    result = utils.valid_recipe_file_size('small_recipe.json')
+    assert result
```

Comment:
Nit: create/mock a file with contents size less than 16KB and then test if it is valid. Same for the test below.